pixi/backend/api.js

```diff
@@ -72,6 +74,7 @@ const execChecks = async (url) => {
     components: findAmpComponents($),
     data: lintResults,
     ...result,
+    version: API_VERSION
```

Comment:
You do not need that here, since the version is already added after the `await execChecks` below.